.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Description
```

Comment:
this actually adds the pull request template with generative AI field to see if the user creates the PR by generative AI. i think this pull request can be general use, so applying this to entire organization should be fine.